.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: true
```

Comment:
What does this do?

Reply:
Allows users not to use any templates at all [documentation](https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

Reply:
Ah, neat!